exercises/raindrops/instructions.md

```diff
@@ -16,7 +16,8 @@ If a given number:
 - 34 is not divisible by 3, 5, or 7, so the result would be `"34"`.
 
 ~~~~exercism/note
-The simplest way to test if one number is evenly divisible by another is to use the [remainder][remainder] or [modulo][modulo] operator.
+A common way to test if one number is evenly divisible by another is to compare the [remainder][remainder] or [modulus][modulo] to zero.
+Most languages provide operators or functions for one (or both) of these.
```

Comment:
This looks better, I was going to comment about it being "simplest" as, well, it is gone now, and this is better.